packages/syndic/syndic.1.6.0/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+name:         "syndic"
+maintainer:   "Romain Calascibetta <romain.calascibetta@gmail.com>"
+authors:      [ "Romain Calascibetta"
+                "Christophe Troestler" ]
+license:      "MIT"
+homepage:     "https://github.com/Cumulus/Syndic"
+dev-repo:     "git+https://github.com/Cumulus/Syndic.git"
+bug-reports:  "https://github.com/Cumulus/Syndic/issues"
+doc:          "https://cumulus.github.io/Syndic/"
+synopsis:     "RSS1, RSS2, Atom and OPML1 parsing"
+description: """
+Pure OCaml Library for parsing and writing various types of
+feeds and subscriber lists."""
+
+build: [
+  [ "dune" "subst" ]
```

Comment:
do you need to call `dune subst` every time? It's usually only required if you pin the package, i.e. `{pinned}`